fixlib/fixlib/baseresources.py

```diff
@@ -1453,6 +1471,8 @@ class BaseDNSZone(BaseResource):
     _kind_description: ClassVar[str] = "A DNS zone."
     _metadata: ClassVar[Dict[str, Any]] = {"icon": "dns", "group": "networking"}
     _categories: ClassVar[List[Category]] = [Category.dns, Category.networking]
+    private_zone: Optional[bool] = None
+    zone_resource_record_set_count: Optional[int] = field(default=None, metadata=dict(ignore_history=True))
```

Comment:
```suggestion
    record_count: Optional[int] = field(default=None, metadata=dict(ignore_history=True))
```
The number of records in the record set?

Reply:
for aws:
```
ResourceRecordSetCount (integer) –

The number of resource record sets in the hosted zone.
```
for azure
```
max_number_of_record_sets

'The maximum number of record sets that can be created in this DNS zone. This is a read-only property and any attempt to set this value will be ignored.
```